superset-frontend/src/dashboard/constants.ts

```diff
@@ -37,6 +37,7 @@ export const PLACEHOLDER_DATASOURCE: Datasource = {
 export const MAIN_HEADER_HEIGHT = 53;
 export const CLOSED_FILTER_BAR_WIDTH = 32;
 export const OPEN_FILTER_BAR_WIDTH = 260;
```

Comment:
Maybe rename this to `OPEN_FILTER_BAR_DEFAULT_WIDTH`?

Reply:
I kept this variable because some custom override might be impacted.